cypress/pageobject/Users/UserCreation.ts

```diff
@@ -1,65 +1,106 @@
 // UserCreation.ts
 export class UserCreationPage {
-  clickElementById(elementId: string) {
-    cy.get("#" + elementId).click();
+  clickProfileName() {
+    cy.get("#user-profile-name").click();
   }
-
-  typeIntoElementById(elementId: string, value: string) {
-    cy.get("#" + elementId)
-      .click()
-      .type(value);
+  clickProfileButton() {
```

Comment:
```suggestion
  clickProfileButton() {
```

- reusage the navigateToProfile() in the ManageUserPage.ts file, recheck for existing function in the page object file under the Users Folder

Reply:
Used the navigateToProfile()